README.md

```diff
@@ -57,7 +57,7 @@ The source code can be found at https://github.com/kiwix/kiwix-js
 
 ## Unit tests
 
-Unit tests can be run by opening tests.html file on Firefox or Edge (or Chromium/Chrome with some tweaks).
+Unit tests can be run by opening `tests/index.html` file in Firefox, Edge, or Chromium/Chrome.
```

Comment:
The way I've run these ad-hoc is by executing `php -S localhost:4000` or `python3 -m http.server 4000` to start a static server from the repository directory. These are (while using somewhat unrelated technologies) are rather ubiquitous, simple, and dependency-free.

Reply:
I removed `"some tweaks"` since I encountered no issue running these in Chrome. I'm not sure what these tweaks referred to, but I guess there was an issue at some point that required modifying some of the files?

I'm happy to revert this change. Did I misunderstand?

Reply:
I think the tweaks referred to running the file from file://  (--allowfileaccessfromfile or something like that), but this has largely been blocked by all but legacy browsers. So it's fine to remove the reference to tweaks.

Reply:
I agree to remove the `tweaks` reference, but we should mention here what is necessary to do for a newcomer : if a `npm install` is necessary or not, and that it must be opened through a local webserver (and suggest the php and python ways)

Reply:
If `npm install` is now necessary, it also should be mentioned at the beginning of the body of tests/index.html